docs/deploy.md

```diff
@@ -68,8 +68,8 @@ qgb deploy \
 
 The `latest` can be replaced by the following:
 
-- `latest`: to deploy the QGB contract starting from the latest validator set.
-- `earliest`: to deploy the QGB contract starting from genesis.
-- `nonce`: you can provide a custom nonce on where you want the QGB to start. If the provided nonce is not a `Valset` attestation, then the one before it will be used to deploy the QGB smart contract.
+- `latest`: to deploy the Blobstream contract starting from the latest validator set.
+- `earliest`: to deploy the Blobstream contract starting from genesis.
+- `nonce`: you can provide a custom nonce on where you want the Blobstream to start. If the provided nonce is not a `Valset` attestation, then the one before it will be used to deploy the Blobstream smart contract.
```

Comment:
```suggestion
- `nonce`: you can provide a custom nonce on where you want Blobstream to start. If the provided nonce is not a `Valset` attestation, then the one before it will be used to deploy the Blobstream smart contract.
```